net/pb/net.proto

```diff
@@ -40,8 +32,8 @@ message Log {
 
 // GetLogsRequest is used to request thread logs.
 message GetLogsRequest {
-    // header is the message header.
-    Header header = 1;
+    // this was the message header.
+    reserved 1;
```

Comment:
For clarity it's better to just move the fields from the `body` field to top level, but I'm not sure if we ever want to allow old and new code to co-exist for a while.

Reply:
Agreed! Let's just leave the `Body` field for now. We can remove it when we do a [protocol bump](https://github.com/textileio/go-threads/blob/master/core/thread/protocol.go#L17). The protocol tag is used by libp2p to differentiate p2p services.